src/llmq/dkgsessionhandler.cpp

```
@@ -4,47 +4,29 @@
-#include <util/underlying.h>
-
-namespace llmq
-{
 
+namespace llmq {
 CDKGSessionHandler::CDKGSessionHandler(CBLSWorker& _blsWorker, CDeterministicMNManager& dmnman,
                                        CDKGDebugManager& _dkgDebugManager, CDKGSessionManager& _dkgManager,
-                                       CMasternodeMetaMan& mn_metaman, CQuorumBlockProcessor& _quorumBlockProcessor,
-                                       CQuorumSnapshotManager& qsnapman, const CActiveMasternodeManager* const mn_activeman,
-                                       const ChainstateManager& chainman, const CSporkManager& sporkman,
+                                       CQuorumSnapshotManager& qsnapman, const ChainstateManager& chainman,
```

Cppcheck (2.19.0)

[error] #error No known always_inline attribute for this platform.

(preprocessorErrorDirective)